Orange/widgets/visualize/owscatterplot.py

```diff
@@ -179,22 +182,36 @@ def __init__(self):
         self.sampling.setVisible(False)
 
         box = gui.vBox(self.controlArea, "Points")
+        color_model = DomainModel(
+            order=("(Same color)", dmod.Separator) + dmod.SEPARATED,
+            valid_types=dmod.PRIMITIVE)
         self.cb_attr_color = gui.comboBox(
             box, self, "graph.attr_color", label="Color:",
             emptyString="(Same color)", callback=self.update_colors,
```

Comment:
Is there any way this `emptyString` wouldn't need to be redefined? It's defined in the model, that should be enough and should work then with any users of DomainModel, not just combos.

Like:
```py
model = DomainModel(..., default='(Same color)')  # default is always first
# and also
assert model.default == '(Same color)'
```
and the gui "front" handlers can check:
```py
...
value = model.data(index)
return value if value is not model.default else None